packages/plugins/users-permissions/server/controllers/validation/auth.js

```diff
@@ -27,31 +32,46 @@ const forgotPasswordSchema = yup
   })
   .noUnknown();
 
-const resetPasswordSchema = yup
-  .object({
-    password: yup.string().required(),
-    passwordConfirmation: yup.string().required(),
-    code: yup.string().required(),
-  })
-  .noUnknown();
+const resetPasswordSchema = (config) =>
+  yup
+    .object({
+      password: yup
+        .string()
+        .required()
+        .min(config?.password?.min || 6)
```

Comment:
adding this as a default would be a Breaking change

Reply:
hmmmm, sure we can make not the default
we can use a custom yup schema but that would make it a breaking change if we change yup later on

Reply:
Yeaha agreed, I was thinking more about a function so we can just pass it the password and the user can return true / false or throw an Error for ex

Reply:
oh yeah even better i was thinking to add a method like this specifically if they wanted to add a specific password pattern, but yeah it would work to maintain also the min, max value too
would you suggest removing the min max, and just providing them a method ? or keeping both options

Reply:
I would do just a function so we can wrap it in a yup.test()